Frends.SFTP.MoveFile/Frends.SFTP.MoveFile/Frends.SFTP.MoveFile.cs

```diff
@@ -11,20 +11,20 @@ namespace Frends.SFTP.MoveFile;
 public class SFTP
 {
     /// <summary>
-    /// Moves a file in SFTP server.
+    /// Moves files in SFTP server.
     /// [Documentation](https://tasks.frends.com/tasks/frends-tasks/Frends.SFTP.MoveFile)
     /// </summary>
     /// <param name="connection">Transfer connection parameters</param>
     /// <param name="input">Read options with full path and encoding</param>
     /// <param name="cancellationToken">CancellationToken given by Frends.</param>
-    /// <returns>Result object { List [object { string SourcePath, string TargetPath }, string message] }</returns>
-    public static Result MoveFile([PropertyTab] Input input, [PropertyTab] Connection connection, CancellationToken cancellationToken)
+    /// <returns>Object { List&lt;FileItem&gt; [ { string SourcePath, string TargetPath } ], string message }</returns>

```

Comment:
Please also add property name `Files`, since you have to refer to it to be able to use it.
`message` --> `Message`
`TargetPath` --> `DestinationPath`